x-pack/plugins/cases/public/components/custom_fields/index.tsx

```diff
@@ -51,6 +53,14 @@ const CustomFieldsComponent: React.FC<Props> = ({
     setError(false);
   }, [handleAddCustomField, setError, customFields, error]);
 
+  const onEditCustomField = useCallback(
```

Comment:
Could you please add a test in `x-pack/plugins/cases/public/components/custom_fields/index.test.tsx` to test the new logic?

Reply:
added in [20335ce](https://github.com/elastic/kibana/pull/167386/commits/20335cebb57a6af581ec87db8f371fdfb9bbd30e#diff-7fbbd56df09646a50634057908c1865e282a72cf4ee4f59e8245ea52eb404895)